plover/dictionary/test_loading_manager.py

```diff
@@ -25,8 +25,6 @@ def __call__(self, filename):
         loader = MockLoader(files)
         with patch('plover.dictionary.loading_manager.load_dictionary', loader):
             manager = loading_manager.DictionaryLoadingManager()
-            manager.start_loading('a')
```

Comment:
This line tests that the start_loading function works on its own. The other dictionaries are loaded with the load function.